ndbench-api/src/main/java/com/netflix/ndbench/api/plugin/builtin/IEsConfig.java

```diff
@@ -0,0 +1,66 @@
+package com.netflix.ndbench.api.plugin.builtin;
+
+
+import com.netflix.archaius.api.annotations.Configuration;
+import com.netflix.archaius.api.annotations.DefaultValue;
+
+@Configuration(prefix = "ndbench.config.es")
+public interface IEsConfig {
```

Comment:
I am in the opinion that we should not overload the `ndbench-api` project with ES specific configuration and that the `IEsConfig` should go under the ES plugin.

Reply:
makes sense..  i sent you and Vinnay email proposal.. will also cc Gennadiy.